keyboards/keychron/c3_pro/info.json

```
@@ -0,0 +1,122 @@
+{
+    "keyboard_name": "Keychron C3 Pro",
+    "manufacturer": "Keychron",
+    "url": "https://github.com/Keychron",
+    "maintainer": "lalalademaxiya1",
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "usb": {
+        "vid": "0x3434"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["C6", "C7", "C8", "A14", "A15", "C10", "C11", "C13", "C14", "C15", "C0", "C1", "C2", "C3", "A0","A1", "A2"],
+        "rows": ["C12", "D2", "B3", "B4", "B5", "B6"]
+    },
+    "diode_direction": "ROW2COL",
+    "layouts": {
```

matches community layout
```suggestion
    "community_layouts: ["tkl_ansi"],
    "layouts": {
```

---

line 24+ of the suggestion here should be
```json
    "community_layouts": ["tkl_ansi"],
```